.gitignore

```diff
@@ -3,7 +3,6 @@
 .bundle
 .config
 .yardoc
-Gemfile.lock
```

Comment:
For a library I think it's a general best practice to not commit the lockfile but rather just have some constraints in the gemspec

Reply:
I'd like to keep it as without it if someone clones our repo they might install different versions of dev environment which might cause problems if they want to improve this gem.